src/components/AboutMeSection/AboutMeSection.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useStaticQuery, graphql } from 'gatsby';
+import Img from 'gatsby-image';
+import { rem } from 'polished';
+import { FormattedMessage } from 'gatsby-plugin-intl';
+
+import { breakPoints } from 'styles/base';
+import { SectionWrapper, SectionHeader } from '../Section/styled';
+
+const StyledImg = styled(Img)`
+  width: 90%;
+  border-radius: ${rem(5)};
+
+  @media (min-width: ${breakPoints.desktop}) {
+    width: 40%;
+    border-radius: ${rem(10)};
+  }
+`;
+
+const SectionText = styled.p`
+  width: 90%;
```

Comment:
you will have problems with this approach. Let's use grid (I am not talking about css-grid, but concept itself - implementation is up to you), but this is for sure wrong.